app/models/computacenter/outgoing_api/cap_update_request.rb

```diff
@@ -49,4 +51,20 @@ def construct_body
   def renderer
     Computacenter::OutgoingAPI::BaseController
   end
+
+private
+
+  def success?
+    @response.status.success? && xml_success?
+  end
+
+  def xml_response
+    @xml_response ||= Nokogiri::XML(@response.body)
+  end
+
+  def xml_success?
+    return true if xml_response.css('HeaderResult').blank?
+
+    xml_response.css('HeaderResult').attr('status').value != 'Failed'
```

Comment:
I'd prefer this to be a positive check on `value == 'Success'` rather than `!= 'Failed'`

Reply:
I've seen the api docs so I've updated this now